src/tests/TransposeMatrix/sources.cmake

```diff
@@ -0,0 +1,3 @@
+add_lab("TransposeMatrix")
+add_lab_solution("TransposeMatrix" ${CMAKE_CURRENT_LIST_DIR}/tranTB.cu)
+#add_generator("MultiplyMatrix")
```

Comment:
see https://github.com/mitchdz/HAMC/blob/master/src/tests/RREFMatrix/sources.cmake#L3

Adding a new source file here such as tranTB.cu will create a file titled "TransposeMatrix_DatasetGenerator" in your build directory (the suffix is derived from the lab name)